Bottom: e0a3fdef1af1e4b3c16c349dfb1e1fd076eb93e4
Top:    319777a721dd67948e411809a2a67862ee3fa764
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 20:07:16 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
